### DISCUSSION TITLE
[Help] Cannot open Chat Panel

### DISCUSSION THREAD

**Discussion by feroze-mohamed** [Genuitec/Copilot4Eclipse-community#4] [Help] [answered]
I installed the plugin from Eclipse Marketplace. 
Login is successful and suggestions are working too. 

but the Chat panel doesnt open :

"An internal error occurred during: "View Readiness Detect Job".
Cannot invoke "java.net.URI.toString()" because "uri" is null"

**Eclipse details:**
Eclipse IDE for C/C++ Developers (includes Incubating components)

Version: 2024-06 (4.32.0)
Build id: 20240606-1231

Copilot4Eclipse Version : 1.2.1
Copilot API Version 1.180.0

**What i Tried :**
1. check for updates and update eclipse and plugins
2. uninstall the plugin and reinstall again

**Comment by wayneparrott** [comment]
@feroze-mohamed Thanks for this report. 

What OS and version?

Can you try starting up Eclipse, open the Chat Panel and then check the Eclipse Error Log View for any interesting entries? 
Specifically looking for any messages relating to the Chat Panel's internal URL or issues with a Jetty webserver not starting.

**Comment by feroze-mohamed** [comment, marked as answer]
Hi Thank you for the quick response. 

Seems like the issue is no lo longer reproducible. 
Not sure what fixed but below are all the things i did:
> updated Eclipse and all plugins
>  uninstalled Copilot4Eclipse, restarted eclipse
>  installed Copilot4Eclipse, restarted eclipse
> rebooted the VMWare (Ubuntu)

And now it seem to be working.
